federation/cmd/federation-apiserver/app/server.go

```diff
@@ -38,6 +38,7 @@ import (
 	"k8s.io/kubernetes/pkg/genericapiserver/authorizer"
 	genericoptions "k8s.io/kubernetes/pkg/genericapiserver/options"
 	genericvalidation "k8s.io/kubernetes/pkg/genericapiserver/validation"
+	"k8s.io/kubernetes/pkg/master/routes"
```

Comment:
We do not want federation to depend on pkg/master which pulls in all kubernetes APIs.
We moved the common things out of pkg/master into pkg/genericapiserver